src/evict/evict_lru.c

```diff
@@ -1662,6 +1662,21 @@ __evict_walk_file(WT_SESSION_IMPL *session,
 		FLD_SET(walk_flags, WT_READ_PREV);
```

Comment:
@keithbostic I think we probably also want to revisit ``evict_walk_reverse`` - I suspect we only want to set a walk direction when setting from a NULL evict_ref.